modules/local/download_from_fire.nf

```diff
@@ -0,0 +1,49 @@
+process DOWNLOAD_FROM_FIRE {
+
+    secret 'FIRE_ACCESS_KEY'
+    secret 'FIRE_SECRET_KEY'
+
+    tag "${meta.id}"
+
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'oras://community.wave.seqera.io/library/boto3:1.35.37--a82b4d378d332259' :
+        'community.wave.seqera.io/library/pip_boto3:501beb4bd409b3e1' }"
```

Comment:
what I see!

Reply:
Seqera container for the win... I hope to remove this process in the near future so, I thought seqera containers is a good enough use case for this